alacritty_terminal/src/tty/windows/mod.rs

```diff
@@ -45,6 +45,8 @@ pub enum PtyHandle {
 }
 
 pub struct Pty {
+    // It is important for drop order that this handle is defined before conout. Drop for Conpty
+    // will deadlock if the conout pipe has already been dropped.
```

Comment:
I'd suggest the following:

```
// XXX: Handle is required to be the first field, to ensure correct drop order. Dropping `conout` before `handle` will cause a deadlock.
```

This should give us the nice benefit to highlight this message in most editors.